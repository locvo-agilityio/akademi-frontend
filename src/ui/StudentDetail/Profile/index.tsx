import { Avatar, Box, Flex, Image, Text, VStack } from '@chakra-ui/react';

// Images
import BgStudent from '@/images/bg-student.webp';

// Components
import Contact from '../Contact';

interface IProfileProps {
  email?: string;
  phone?: string;
  address?: string;
  parentName?: string;
  avatar?: string;
}

const Profile = ({
  email = '',
  phone = '',
  address = '',
  parentName = '',
  avatar = '',
}: IProfileProps) => (
  <VStack
    w="full"
    flexDirection="column"
    minH={498}
    borderRadius="2xl"
    bgColor="white"
    justifyContent="space-between"
  >
    <Box
      w="full"
      h={200}
      position="relative"
      bgColor="primary"
      borderTopRadius="2xl"
    >
      <Image
        w="full"
        h={200}
        objectFit="cover"
        objectPosition="right"
        src={BgStudent}
        alt="Background Student"
      />

      <Box position="absolute" top="50%" left={8} w="full">
        <Flex
          w={160}
          h={160}
          borderRadius="full"
          bgColor="white"
          align="center"
          justify="center"
        >
          <Avatar w="full" h="full" p={2} src={avatar} />
        </Flex>

        <Text as="h2" size="lg" mt={6} fontWeight="bold" color="darkBlue">
          {parentName}
        </Text>
        <Text as="p" mt={2} color="whiteSmoke">
          Student
        </Text>
      </Box>
    </Box>

    <Contact
      email={email}
      parentName={parentName}
      address={address}
      phone={phone}
    />
  </VStack>
);

export default Profile;
