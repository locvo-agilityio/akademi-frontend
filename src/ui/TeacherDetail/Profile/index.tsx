'use client';

import { memo } from 'react';
import { Avatar, Box, Flex, Image, Text, VStack } from '@chakra-ui/react';

// Images
import BgTeacher from '@/images/bg-teacher.webp';

// Components
import Contact from '../Contact';

// Components
import Information from '../Information';

interface IProfileTeacherProps {
  name?: string;
  email?: string;
  phone?: string;
  subject?: string;
  avatar?: string;
  address?: string;
  description?: string;
  degree?: string;
  university?: string;
  expertise?: string;
  startEducation?: string;
  endEducation?: string;
}

const ProfileTeacher = ({
  name = '',
  email = '',
  phone = '',
  subject = '',
  avatar = '',
  address = '',
  description = '',
  degree = '',
  university = '',
  expertise = '',
  startEducation = '',
  endEducation = '',
}: IProfileTeacherProps) => (
  <VStack
    w="full"
    flexDirection="column"
    borderRadius="2xl"
    bgColor="white"
    justifyContent="space-between"
    alignItems="flex-start"
  >
    <Box
      w="full"
      h={200}
      position="relative"
      bgColor="primary"
      borderTopRadius="2xl"
    >
      <Image
        w="full"
        h={200}
        objectFit="cover"
        objectPosition="right"
        src={BgTeacher}
        alt="Background Student"
      />

      <Box position="absolute" maxW={200} top="50%" left={8} w="full">
        <Flex
          w={160}
          h={160}
          borderRadius="full"
          bgColor="white"
          align="center"
          justify="center"
        >
          <Avatar w="full" h="full" p={2} src={avatar} />
        </Flex>

        <Text as="h2" size="lg" mt={6} fontWeight="bold" color="darkBlue">
          {name}
        </Text>
        <Text as="p" mt={2} color="whiteSmoke" fontWeight="semibold">
          {subject} Teacher
        </Text>
      </Box>
    </Box>

    <Flex mt={160} p={8} w="full" flexDirection="column">
      <Contact email={email} address={address} phone={phone} />

      <Information
        degree={degree}
        university={university}
        expertise={expertise}
        description={description}
        startEducation={startEducation}
        endEducation={endEducation}
      />
    </Flex>
  </VStack>
);

export default memo(ProfileTeacher);
