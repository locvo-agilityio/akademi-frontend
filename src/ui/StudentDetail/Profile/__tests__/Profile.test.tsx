import { render } from '@testing-library/react';

import Profile from '..';

describe('Profile', () => {
  it('should snapshot default value', () => {
    const { container } = render(<Profile />);

    expect(container).toBeInTheDocument();
  });

  it('should snapshot successfully', () => {
    const { container } = render(
      <Profile
        address="Jakarta"
        email="john.doe@example.com"
        phone="08123456789"
        parentName="John Doe"
      />,
    );

    expect(container).toMatchSnapshot();
  });
});
