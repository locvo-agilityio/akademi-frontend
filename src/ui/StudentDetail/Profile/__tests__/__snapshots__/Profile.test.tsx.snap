// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should snapshot successfully 1`] = `
<div>
  <div
    class="chakra-stack css-1xfsfgr"
  >
    <div
      class="css-9npxv6"
    >
      <img
        alt="Background Student"
        class="chakra-image css-1m8o377"
        src="test-file-stub"
      />
      <div
        class="css-14ciym9"
      >
        <div
          class="css-1367w0v"
        >
          <span
            class="chakra-avatar css-131besb"
          >
            <svg
              aria-label=" avatar"
              class="chakra-avatar__svg css-16ite8i"
              role="img"
              viewBox="0 0 128 128"
            >
              <path
                d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                fill="currentColor"
              />
              <path
                d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                fill="currentColor"
              />
            </svg>
          </span>
        </div>
        <h2
          class="chakra-text css-hn8sh4"
        >
          John Doe
        </h2>
        <p
          class="chakra-text css-12891xg"
        >
          Student
        </p>
      </div>
    </div>
    <div
      class="css-w87r8w"
    >
      <div
        class="css-1o5j150"
      >
        <p
          class="chakra-text css-cif1vy"
        >
          Parents:
        </p>
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="40"
              viewBox="0 0 40 40"
              width="40"
            >
              <path
                d="M26.176 10.8749C26.176 11.2187 26.1525 11.5624 26.1096 11.9023C26.1291 11.7656 26.1486 11.6249 26.1643 11.4882C26.0705 12.1679 25.8908 12.8281 25.6291 13.4609C25.6799 13.3359 25.7346 13.2109 25.7854 13.0859C25.5236 13.7031 25.1877 14.2812 24.7775 14.8124C24.8596 14.707 24.9416 14.6015 25.0236 14.496C24.6174 15.0195 24.1447 15.4921 23.6213 15.8984C23.7268 15.8163 23.8322 15.7343 23.9377 15.6523C23.4065 16.0585 22.8283 16.3984 22.2111 16.6601C22.3361 16.6093 22.4611 16.5546 22.5861 16.5038C21.9533 16.7656 21.2932 16.9452 20.6135 17.039C20.7502 17.0195 20.8908 16.9999 21.0275 16.9843C20.3439 17.0741 19.6564 17.0741 18.9729 16.9843C19.1096 17.0038 19.2502 17.0234 19.3869 17.039C18.7072 16.9452 18.0471 16.7656 17.4143 16.5038C17.5393 16.5546 17.6643 16.6093 17.7893 16.6601C17.1721 16.3984 16.5939 16.0624 16.0627 15.6523C16.1682 15.7343 16.2736 15.8163 16.3791 15.8984C15.8557 15.4921 15.383 15.0195 14.9768 14.496C15.0588 14.6015 15.1408 14.707 15.2228 14.8124C14.8166 14.2812 14.4768 13.7031 14.215 13.0859C14.2658 13.2109 14.3205 13.3359 14.3713 13.4609C14.1096 12.8281 13.9299 12.1679 13.8361 11.4882C13.8557 11.6249 13.8752 11.7656 13.8908 11.9023C13.801 11.2187 13.801 10.5312 13.8908 9.84759C13.8713 9.98431 13.8518 10.1249 13.8361 10.2617C13.9299 9.58196 14.1096 8.92181 14.3713 8.28899C14.3205 8.41399 14.2658 8.53899 14.215 8.66399C14.4768 8.04681 14.8127 7.46868 15.2228 6.93743C15.1408 7.0429 15.0588 7.14837 14.9768 7.25384C15.383 6.7304 15.8557 6.25774 16.3791 5.85149C16.2736 5.93352 16.1682 6.01556 16.0627 6.09759C16.5939 5.69134 17.1721 5.35149 17.7893 5.08977C17.6643 5.14056 17.5393 5.19524 17.4143 5.24602C18.0471 4.98431 18.7072 4.80462 19.3869 4.71087C19.2502 4.7304 19.1096 4.74993 18.9729 4.76556C19.6564 4.67571 20.3439 4.67571 21.0275 4.76556C20.8908 4.74602 20.7502 4.72649 20.6135 4.71087C21.2932 4.80462 21.9533 4.98431 22.5861 5.24602C22.4611 5.19524 22.3361 5.14056 22.2111 5.08977C22.8283 5.35149 23.4065 5.68743 23.9377 6.09759C23.8322 6.01556 23.7268 5.93352 23.6213 5.85149C24.1447 6.25774 24.6174 6.7304 25.0236 7.25384C24.9416 7.14837 24.8596 7.0429 24.7775 6.93743C25.1838 7.46868 25.5236 8.04681 25.7854 8.66399C25.7346 8.53899 25.6799 8.41399 25.6291 8.28899C25.8908 8.92181 26.0705 9.58196 26.1643 10.2617C26.1447 10.1249 26.1252 9.98431 26.1096 9.84759C26.1525 10.1874 26.1721 10.5312 26.176 10.8749C26.1799 11.6913 26.8908 12.4765 27.7385 12.4374C28.5822 12.3984 29.3049 11.7499 29.301 10.8749C29.2932 9.00384 28.7307 7.08977 27.6291 5.56634C27.3322 5.15618 27.0158 4.75384 26.6643 4.39056C26.3088 4.02337 25.9182 3.70696 25.5119 3.39837C24.7736 2.83587 23.9494 2.42181 23.0744 2.10149C19.5471 0.808525 15.2932 1.96087 12.9064 4.86712C12.5783 5.26556 12.2658 5.67962 12.0002 6.12493C11.7346 6.56634 11.5275 7.03509 11.3361 7.51165C10.9768 8.39446 10.7932 9.33196 10.7189 10.2812C10.5744 12.1406 11.0588 14.082 12.0275 15.6757C12.9611 17.2148 14.3713 18.5351 16.0197 19.2812C16.5041 19.4999 17.0002 19.7031 17.5158 19.8476C18.0275 19.9882 18.5471 20.0663 19.0744 20.1288C20.0354 20.2421 21.0158 20.164 21.965 19.9726C25.5705 19.2382 28.59 16.1327 29.1486 12.4882C29.2307 11.957 29.2932 11.4257 29.2932 10.8867C29.2971 10.0702 28.5705 9.28509 27.7307 9.32415C26.8869 9.35149 26.176 9.99993 26.176 10.8749ZM31.3596 35.3007C30.594 35.3007 29.8283 35.3007 29.0627 35.3007C27.2385 35.3007 25.4182 35.3007 23.594 35.3007C21.3986 35.3007 19.2033 35.3007 17.0041 35.3007C15.1096 35.3007 13.215 35.3007 11.3166 35.3007C10.4299 35.3007 9.54316 35.3085 8.65644 35.3007C8.55878 35.3007 8.46112 35.2929 8.36737 35.2812C8.50409 35.3007 8.64472 35.3202 8.78144 35.3359C8.62519 35.3124 8.47675 35.2695 8.33222 35.2109C8.45722 35.2617 8.58222 35.3163 8.70722 35.3671C8.55097 35.3007 8.40644 35.2148 8.26972 35.1093C8.37519 35.1913 8.48065 35.2734 8.58612 35.3554C8.46894 35.2577 8.35956 35.1523 8.26581 35.0351C8.34784 35.1406 8.42987 35.246 8.5119 35.3515C8.40644 35.2148 8.3244 35.0702 8.25409 34.914C8.30487 35.039 8.35956 35.164 8.41034 35.289C8.35175 35.1445 8.31269 34.9921 8.28534 34.8398C8.30487 34.9765 8.3244 35.1171 8.34003 35.2538C8.27753 34.7812 8.3205 34.2812 8.3205 33.8046C8.3205 33.246 8.3205 32.6913 8.3205 32.1327C8.3205 31.7148 8.34394 31.3007 8.39862 30.8867C8.37909 31.0234 8.35956 31.164 8.34394 31.3007C8.45331 30.4999 8.66425 29.7148 8.98065 28.9687C8.92987 29.0937 8.87519 29.2187 8.8244 29.3437C9.12909 28.6327 9.51972 27.9648 9.99237 27.3515C9.91034 27.457 9.82831 27.5624 9.74628 27.6679C10.2189 27.0624 10.7619 26.5156 11.3713 26.0429C11.2658 26.1249 11.1603 26.207 11.0549 26.289C11.6682 25.8163 12.3361 25.4257 13.0471 25.121C12.9221 25.1718 12.7971 25.2265 12.6721 25.2773C13.4182 24.9648 14.1994 24.7499 15.0041 24.6406C14.8674 24.6601 14.7268 24.6796 14.59 24.6952C15.1682 24.621 15.7424 24.6171 16.3244 24.6171C17.0393 24.6171 17.7541 24.6171 18.4689 24.6171C20.1369 24.6171 21.8049 24.6171 23.4729 24.6171C24.1174 24.6171 24.758 24.6132 25.4025 24.6952C25.2658 24.6757 25.1252 24.6562 24.9885 24.6406C25.7893 24.7499 26.5744 24.9609 27.3205 25.2773C27.1955 25.2265 27.0705 25.1718 26.9455 25.121C27.6565 25.4257 28.3244 25.8163 28.9377 26.289C28.8322 26.207 28.7268 26.1249 28.6213 26.0429C29.2268 26.5156 29.7736 27.0585 30.2463 27.6679C30.1643 27.5624 30.0822 27.457 30.0002 27.3515C30.4729 27.9648 30.8635 28.6327 31.1682 29.3437C31.1174 29.2187 31.0627 29.0937 31.0119 28.9687C31.3244 29.7148 31.5393 30.496 31.6486 31.3007C31.6291 31.164 31.6096 31.0234 31.594 30.8867C31.6682 31.4765 31.6721 32.0624 31.6721 32.6562C31.6721 33.3007 31.6721 33.9452 31.6721 34.5898C31.6721 34.8124 31.6799 35.0351 31.6525 35.2538C31.6721 35.1171 31.6916 34.9765 31.7072 34.8398C31.6838 34.996 31.6408 35.1445 31.5822 35.289C31.633 35.164 31.6877 35.039 31.7385 34.914C31.6721 35.0702 31.5861 35.2148 31.4807 35.3515C31.5627 35.246 31.6447 35.1406 31.7268 35.0351C31.6291 35.1523 31.5236 35.2616 31.4065 35.3554C31.5119 35.2734 31.6174 35.1913 31.7229 35.1093C31.5861 35.2148 31.4416 35.2968 31.2854 35.3671C31.4104 35.3163 31.5354 35.2617 31.6604 35.2109C31.5158 35.2695 31.3635 35.3085 31.2111 35.3359C31.3479 35.3163 31.4885 35.2968 31.6252 35.2812C31.5393 35.2929 31.4494 35.2968 31.3596 35.3007C30.9572 35.3046 30.5432 35.4726 30.2541 35.7577C29.9846 36.0273 29.7775 36.4726 29.7971 36.8632C29.8361 37.6992 30.4846 38.4374 31.3596 38.4257C32.8791 38.4023 34.2151 37.4101 34.6604 35.9531C34.8361 35.3749 34.7971 34.7538 34.7971 34.1601C34.7971 32.8007 34.8401 31.4531 34.6057 30.1054C34.2619 28.1327 33.2697 26.2773 31.9104 24.8202C30.551 23.3632 28.7229 22.2929 26.7893 21.8163C25.6799 21.5429 24.5627 21.4999 23.4299 21.4999C22.2658 21.4999 21.1057 21.4999 19.9416 21.4999C18.7932 21.4999 17.6486 21.4999 16.5002 21.4999C15.34 21.4999 14.1877 21.5546 13.0588 21.8554C11.1408 22.3632 9.33612 23.4452 8.00409 24.914C6.66034 26.3945 5.68768 28.2538 5.37128 30.2382C5.16034 31.5702 5.20331 32.9062 5.20331 34.2499C5.20331 34.8671 5.16815 35.5117 5.39081 36.1015C5.67987 36.8593 6.14081 37.457 6.81659 37.9062C7.34393 38.2538 8.0119 38.4218 8.6369 38.4296C8.94159 38.4335 9.24628 38.4296 9.55097 38.4296C10.3322 38.4296 11.1096 38.4296 11.8908 38.4296C13.0002 38.4296 14.1057 38.4296 15.215 38.4296C16.5041 38.4296 17.7932 38.4296 19.0822 38.4296C20.4064 38.4296 21.7268 38.4296 23.051 38.4296C24.2619 38.4296 25.469 38.4296 26.6799 38.4296C27.6291 38.4296 28.5783 38.4296 29.5236 38.4296C30.0666 38.4296 30.6096 38.4296 31.1525 38.4296C31.2229 38.4296 31.2932 38.4296 31.3635 38.4296C32.1799 38.4296 32.9651 37.7109 32.926 36.8671C32.8869 36.0156 32.2385 35.3007 31.3596 35.3007Z"
                fill="currentColor"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          >
            John Doe
          </p>
        </div>
      </div>
      <div
        class="css-1o5j150"
      >
        <p
          class="chakra-text css-cif1vy"
        >
          Address:
        </p>
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="20"
              viewBox="0 0 16 20"
              width="16"
            >
              <path
                d="M8 0C5.87827 0 3.84344 0.842855 2.34315 2.34315C0.842855 3.84344 0 5.87827 0 8C0 13.4 7.05 19.5 7.35 19.76C7.53113 19.9149 7.76165 20.0001 8 20.0001C8.23835 20.0001 8.46887 19.9149 8.65 19.76C9 19.5 16 13.4 16 8C16 5.87827 15.1571 3.84344 13.6569 2.34315C12.1566 0.842855 10.1217 0 8 0ZM8 17.65C5.87 15.65 2 11.34 2 8C2 6.4087 2.63214 4.88258 3.75736 3.75736C4.88258 2.63214 6.4087 2 8 2C9.5913 2 11.1174 2.63214 12.2426 3.75736C13.3679 4.88258 14 6.4087 14 8C14 11.34 10.13 15.66 8 17.65ZM8 4C7.20887 4 6.43552 4.2346 5.77772 4.67412C5.11992 5.11365 4.60723 5.73836 4.30448 6.46927C4.00173 7.20017 3.92252 8.00444 4.07686 8.78036C4.2312 9.55628 4.61216 10.269 5.17157 10.8284C5.73098 11.3878 6.44371 11.7688 7.21964 11.9231C7.99556 12.0775 8.79983 11.9983 9.53073 11.6955C10.2616 11.3928 10.8864 10.8801 11.3259 10.2223C11.7654 9.56448 12 8.79113 12 8C12 6.93913 11.5786 5.92172 10.8284 5.17157C10.0783 4.42143 9.06087 4 8 4ZM8 10C7.60444 10 7.21776 9.8827 6.88886 9.66294C6.55996 9.44318 6.30362 9.13082 6.15224 8.76537C6.00087 8.39991 5.96126 7.99778 6.03843 7.60982C6.1156 7.22186 6.30608 6.86549 6.58579 6.58579C6.86549 6.30608 7.22186 6.1156 7.60982 6.03843C7.99778 5.96126 8.39991 6.00087 8.76537 6.15224C9.13082 6.30362 9.44318 6.55996 9.66294 6.88886C9.8827 7.21776 10 7.60444 10 8C10 8.53043 9.78929 9.03914 9.41421 9.41421C9.03914 9.78929 8.53043 10 8 10Z"
                fill="white"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          >
            Jakarta
          </p>
        </div>
      </div>
      <div
        class="css-1o5j150"
      >
        <p
          class="chakra-text css-cif1vy"
        >
          Phone:
        </p>
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="20"
              viewBox="0 0 21 20"
              width="21"
            >
              <path
                d="M18.4401 11.0001C18.2201 11.0001 17.9901 10.9301 17.7701 10.8801C17.3246 10.7819 16.8868 10.6516 16.4601 10.4901C15.9962 10.3213 15.4862 10.3301 15.0284 10.5147C14.5706 10.6993 14.1972 11.0467 13.9801 11.4901L13.7601 11.9401C12.7861 11.3983 11.8911 10.7253 11.1001 9.94011C10.3149 9.14913 9.64192 8.25411 9.1001 7.28011L9.5201 7.00011C9.96347 6.78303 10.3109 6.40964 10.4955 5.9518C10.6801 5.49396 10.6889 4.98402 10.5201 4.52011C10.3613 4.09253 10.231 3.65491 10.1301 3.21011C10.0801 2.99011 10.0401 2.76011 10.0101 2.53011C9.88867 1.82573 9.51973 1.18785 8.96972 0.731352C8.41972 0.274849 7.7248 0.0297211 7.0101 0.0401122H4.0101C3.57913 0.0360657 3.15235 0.124926 2.75881 0.300643C2.36527 0.47636 2.01421 0.734808 1.72953 1.05839C1.44485 1.38198 1.23324 1.76311 1.10909 2.17583C0.984942 2.58855 0.95118 3.02317 1.0101 3.45011C1.54284 7.63949 3.45613 11.532 6.44775 14.5127C9.43938 17.4935 13.3388 19.3926 17.5301 19.9101H17.9101C18.6475 19.9112 19.3595 19.6406 19.9101 19.1501C20.2265 18.8672 20.4792 18.5203 20.6516 18.1324C20.8239 17.7446 20.9121 17.3245 20.9101 16.9001V13.9001C20.8979 13.2055 20.6449 12.5367 20.1944 12.0078C19.744 11.4789 19.1239 11.1227 18.4401 11.0001ZM18.9401 17.0001C18.9399 17.1421 18.9095 17.2824 18.8509 17.4117C18.7923 17.5411 18.7068 17.6564 18.6001 17.7501C18.4884 17.8466 18.3577 17.9187 18.2165 17.9617C18.0753 18.0048 17.9267 18.0178 17.7801 18.0001C14.035 17.5199 10.5563 15.8066 7.89282 13.1304C5.2293 10.4542 3.53251 6.96745 3.0701 3.22011C3.05419 3.07363 3.06813 2.92544 3.1111 2.7845C3.15407 2.64356 3.22517 2.5128 3.3201 2.40011C3.41381 2.29344 3.52916 2.20795 3.65848 2.14933C3.7878 2.09071 3.92812 2.06029 4.0701 2.06011H7.0701C7.30265 2.05494 7.52972 2.13099 7.71224 2.27518C7.89476 2.41937 8.02131 2.62268 8.0701 2.85011C8.1101 3.12345 8.1601 3.39345 8.2201 3.66011C8.33562 4.18726 8.48936 4.70529 8.6801 5.21011L7.2801 5.86011C7.1604 5.91503 7.05272 5.99306 6.96326 6.08971C6.87379 6.18636 6.8043 6.29973 6.75877 6.42331C6.71324 6.54689 6.69257 6.67824 6.69795 6.80983C6.70332 6.94142 6.73464 7.07066 6.7901 7.19011C8.2293 10.2729 10.7073 12.7509 13.7901 14.1901C14.0336 14.2901 14.3066 14.2901 14.5501 14.1901C14.6748 14.1455 14.7894 14.0766 14.8873 13.9873C14.9851 13.898 15.0643 13.7902 15.1201 13.6701L15.7401 12.2701C16.2571 12.455 16.7847 12.6086 17.3201 12.7301C17.5868 12.7901 17.8568 12.8401 18.1301 12.8801C18.3575 12.9289 18.5608 13.0554 18.705 13.238C18.8492 13.4205 18.9253 13.6476 18.9201 13.8801L18.9401 17.0001Z"
                fill="currentColor"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          >
            08123456789
          </p>
        </div>
      </div>
      <div
        class="css-1o5j150"
      >
        <p
          class="chakra-text css-cif1vy"
        >
          Email:
        </p>
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="16"
              viewBox="0 0 22 16"
              width="22"
            >
              <path
                d="M19 0H3C2.20435 0 1.44129 0.316071 0.87868 0.87868C0.31607 1.44129 0 2.20435 0 3V13C0 13.7956 0.31607 14.5587 0.87868 15.1213C1.44129 15.6839 2.20435 16 3 16H19C19.7956 16 20.5587 15.6839 21.1213 15.1213C21.6839 14.5587 22 13.7956 22 13V3C22 2.20435 21.6839 1.44129 21.1213 0.87868C20.5587 0.316071 19.7956 0 19 0ZM20 12.75L15.1 8.35L20 4.92V12.75ZM2 4.92L6.9 8.35L2 12.75V4.92ZM8.58 9.53L10.43 10.82C10.5974 10.9361 10.7963 10.9984 11 10.9984C11.2037 10.9984 11.4026 10.9361 11.57 10.82L13.42 9.53L18.42 14H3.6L8.58 9.53ZM3 2H19C19.1857 2.00149 19.3674 2.05467 19.5245 2.15358C19.6817 2.25249 19.8083 2.39322 19.89 2.56L11 8.78L2.11 2.56C2.19171 2.39322 2.31826 2.25249 2.47545 2.15358C2.63265 2.05467 2.81428 2.00149 3 2Z"
                fill="currentColor"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          >
            john.doe@example.com
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
